hack/image/build-ansible-image.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# build operator binary and base image
+go build -o test/ansible-operator/ansible-operator test/ansible-operator/cmd/ansible-operator/main.go
```

Comment:
@shawn-hurley @mhrivnak I'm guessing `test/ansible-operator/cmd/ansible-operator/main.go` is not (yet, at least) what we want for the official ansible-operator base image?

Can we pull in the latest changes from the [main.go](https://github.com/water-hole/ansible-operator/blob/master/cmd/manager/main.go) in `github.com/water-hole/ansible-operator` or do we need to wait on #823?

Reply:
I think we can pull in the latest from `water-hole` I don't see any reason not too. Once this change goes in the this becomes our main file I believe. 

Personally, I would like to see these at a new location that is not hidden under the `test` dir. I think that we can deal with that stuff much later, just throwing out the concern.

Reply:
@shawn-hurley For the scorecard PR, I have the image for the scorecard-proxy in a new `images` folder. I think it would be a good idea to move our base images into a specific directory like that. We could make another PR after this gets merged to move the base images to a more reasonable location.

Reply:
@shawn-hurley I reverted my changes to `test/ansible-operator/cmd/ansible-operator/main.go` to make it match water-hole. With #754, the current version looks like what we want. WDYT?